### ISSUE TITLE
Problem/Issues

### ISSUE THREAD

**Issue by jayzkie18** [sologenic/sologenic-expo-wallet#153]
I have an issue in transfer XRP to another wallet, can you help me?

**Comment by jayzkie18** [comment]
Is there is anyone experience problem/ issue transfer of XRP , solo wallet to another external wallet?

**Comment by joshmcl8** [comment]
Yes I have the same problem. Very concerning, hope it gets sorted quickly

**Comment by michaelmcfann26** [comment]
Yes having same problem I’m very concerned as I have a lot of SOLO just stuck on the SOLO DEX I’m starting to get sick about it.

**Comment by jayzkie18** [comment]
Please update sologenic wallet to version 2.2.5, to fix the issue,
available on play store
